apps/etterlatte-samordning-vedtak/README.md

```diff
@@ -71,6 +76,8 @@ Det må foreligge et tjenestepensjonsforhold og -ytelse i Tjenestepensjonsregist
 | 020-VEDTAK-TILGANG      | 403             | Ikke tilgang til vedtak/etterspurt data                  |
 | 021-VEDTAK-FORESPOERSEL | 400             | Feil ved spørring mot vedtak                             |
 | 022-VEDTAK-IKKE-FUNNET  | 404             | Kunne ikke finne vedtaksressurs                          |
+| 030-BEHANDLING-TILGANG  | 404             | Ikke tilgang til sak                                     |
+| 030-BEHANDLING-FORESPOERSEL  | 404             | Feil ved spørring mot behandling                         |
```

Comment:
Trur desse bør ha forskjellige numre, ikkje begge to på 30

Reply:
Enig